seedlet/style-rtl.css

```diff
@@ -8,7 +8,7 @@ Description: Seedlet is a free WordPress theme. A two-column layout and classica
 Requires at least: 5.5
 Tested up to: 5.5
 Requires PHP: 5.6.2
-Version: 1.1.19-wpcom
+Version: 1.1.16-wpcom
```

Comment:
update source `scss` files to a later version.

Reply:
> Where do I update this string and what should I update it to? I see it [here](https://github.com/Automattic/themes/blob/c58ac1b8a8c09401a4874f0f51b4831baf5072c5/seedlet/assets/sass/style.scss#L10) but I'm not sure on how best to do this. 

Thats correct. Please update the line to `Version: 1.1.23-wpcom` in `assets/sass/style.scss` and run `npm run build`

I will make a separate fix to tackle for all the themes in general to avoid inconsistencies in future.

Reply:
Fixed the version conflict.